shared/teams/index.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import TeamsContainer from './render'
+import pausableConnect from '../util/pausable-connect'
+import openURL from '../util/open-url'
+
+import type {RouteProps} from '../route-tree/render-route'
+
+// $FlowIssue type this connector
+export default pausableConnect(
```

Comment:
I copied the settings tab code here. Is there a better example I should be using?

Reply:
I'd split these out into name functions. we use mapStateToProps, mapDispatchToProps

Reply:
Done.